### ISSUE TITLE
Parsing fails when using mutually exclusive arguments

### ISSUE THREAD

**Issue by jazzynico** [s-ball/i18nparse#3]
Reproduced on Windows 7 (via MSYS) with Python 3.6.8, Python 3.9.2, i18nparse 0.7.0.dev1, and the following code:

```
import argparse
import i18nparse

i18nparse.activate()
parser = argparse.ArgumentParser(description="Testing parse error")
mode = parser.add_mutually_exclusive_group(required=True)
mode.add_argument('-f', '--first', dest='first', action='store_true',
                  help='first mode')
mode.add_argument('-s', '--second', dest='second', action='store_true',
                  help='second mode')

args = parser.parse_args()
```
If I use both -f and -s argument (they are mutually exclusive), the application exits with the following tracebacks:

**Python 3.6**
```
Traceback (most recent call last):
  File "./i18nparse_test.py", line 7, in <module>
    i18nparse.activate()
  File "d:\Dev\python\Python34\lib\site-packages\i18nparse\i18nparse.py", line 20, in activate
    [lang], fallback=True)
  File "d:\Dev\python\Python34\Lib\gettext.py", line 400, in translation
    mofiles = find(domain, localedir, languages, all=True)
  File "d:\Dev\python\Python34\Lib\gettext.py", line 372, in find
    for nelang in _expand_lang(lang):
  File "d:\Dev\python\Python34\Lib\gettext.py", line 114, in _expand_lang
    loc = locale.normalize(loc)
  File "d:\Dev\python\Python34\Lib\locale.py", line 390, in normalize
    code = localename.lower()
AttributeError: 'NoneType' object has no attribute 'lower'
```

**Python 3.8**
```
Traceback (most recent call last):
  File "d:\dev\i18nparse_test.py", line 15, in <module>
    args = parser.parse_args()
  File "d:\Dev\python\Python39\Lib\argparse.py", line 1818, in parse_args
    args, argv = self.parse_known_args(args, namespace)
  File "d:\Dev\python\Python39\Lib\argparse.py", line 1851, in parse_known_args
    namespace, args = self._parse_known_args(args, namespace)
  File "d:\Dev\python\Python39\Lib\argparse.py", line 2060, in _parse_known_args
    start_index = consume_optional(start_index)
  File "d:\Dev\python\Python39\Lib\argparse.py", line 2000, in consume_optional
    take_action(action, args, option_string)
  File "d:\Dev\python\Python39\Lib\argparse.py", line 1923, in take_action
    raise ArgumentError(action, msg % action_name)
ValueError: incomplete format
```

No issue when I comment out `i18nparse.activate()`.
No issue with the other argparse messages (--help, argument missing...).

Could it be an issue with the fallback when the original string is not in the locale PO file?

**Comment by s-ball** [comment]
Looks like a bug anyway... Let me try to reproduce and I'll be back in a few days...

**Comment by s-ball** [comment]
I could identify the problem. When you call `i18nparse.activate` **before selecting a locale with `locale.setlocale`** you are implicitely using the portable `C` locale and `locale.getlocale()` which is used in `activate` returns `None`... And this is the cause of the other errors. I have already pushed a quick fix in master that no longer exhibits the error.
I should soon publish a 0.7.1 version containing it.

**Comment by s-ball** [comment]
The problem should be fixed in the 0.7.2 version which has just been uploaded to PyPI
